evelyn_chavez/module_practive/human1/human1.rb

```diff
@@ -0,0 +1,21 @@
+class Human
+    attr_accessor :strength, :intelligence, :stealth, :health
+
+    def initialize
+        @strength = 3
+        @intelligence = 3
+        @stealth = 3
+        @health = 100
+    end
+
+    def attack(object)
+        if object.class.ancestors
```

Comment:
we wanna check if the object is Human. object.class.ancestors gives a list of ancestors.
if object.class.ancestors.include? Human